web-app/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  componentDidMount = () => {
+      // This code should call: http://localhost:3000/api/status
+      // and return {"response":"Everything is working correctly"}
+      fetch('/api/status')
```

Comment:
Why could you not just call over to localhost:5000? did you get a cors error?

Reply:
Yeap! I got a cors error. Also, the problem of using directly dev endpoints is that we'll need to change that once deployed (or implement some logic like: if DEVELOPMENT => localhost:3000, otherwise "ferreiro.me/". I think that's the purpose of the proxy, but not working)

```Access to fetch at 'http://localhost:3000/api/status' from origin 'http://localhost:5000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.```